examples/vsphere/sample-terraform-dhcp.tfvars

```diff
@@ -19,8 +19,8 @@ vsphere_network       = "" # Name of the network to to use for the VMs
 system_ssh_keys_dir = "~/.ssh/oss" # Directory holding public keys to be used on each system
 
 # Kubernetes - Cluster
-cluster_version        = "1.23.8"       # Kubernetes Version
-cluster_cni            = "calico"       # Kuberentes Container Network Interface (CNI)
+cluster_version        = "1.24.10"      # Kubernetes Version
```

Comment:
Is specifying only `major.minor` K8s version not supported?

Reply:
@riragh, not for OSS, since we manage the K8s version and not done by a cloud provider, it's directly downloaded from https://apt.kubernetes.io/ where we need to provide an exact version. 

At this point I chose the latest patch version of 1.24, and since these packages are not ones being managed/patched by a cloud provider and in their "original" form we shouldn't have to worry about them being removed any time soon.

https://packages.cloud.google.com/apt/dists/kubernetes-xenial/main/binary-amd64/Packages